seam/routes/action_attempts.py

```diff
@@ -1,12 +1,29 @@
-from seam.types import AbstractActionAttempts, AbstractSeam as Seam, ActionAttempt
+from seam.types import AbstractSeam as Seam
+from seam.routes.types import AbstractActionAttempts, ActionAttempt
 from typing import Optional, Any, List, Dict, Union
 
 import time
 
-from seam.utils.action_attempt_errors import (
-    SeamActionAttemptFailedError,
-    SeamActionAttemptTimeoutError,
-)
+
+class SeamActionAttemptError(Exception):
```

Comment:
Can we move the errors and special methods out of codegen? We can address this separately from this PR.

Reply:
Are there any other places in the codegen that are "special" like this, i.e., include code or special logic specific to a generated file?

Reply:
I think its only action attempts with their custom errors and polling methods.

Reply:
Relevant issue https://github.com/seamapi/python-next/issues/55